Eastern-Research-Group/expert-query
# Feature/240 uswds v3

## Related Issues:
* [EQ-240](https://jira.epa.gov/browse/EQ-240)

## Main Changes:
* Updated the `uswds` package to version 3, now the `@uswds/uswds` package.
  * Since we aren't compiling in any more Sass, changing the import statements is all that was required.
* Updated the footer to match the most recent EPA template.

## Steps To Test:
1. Open the application, and https://www.epa.gov/themes/epa_theme/pattern-lab/patterns/pages-standalone-template/pages-standalone-template.rendered.html in another tab. Confirm that the Expert Query's page template matches EPA's canonical template. Confirm these changes are reflected in the 400.html and 500.html pages.
2. Go to the application home page, test as many features as possible and verify styles (besides the footer observed above) have not changed.
